fgcm/fgcmFitCycle.py

```diff
@@ -473,36 +498,40 @@ def run(self):
 
 
         # Compute SuperStar Flats
-        self.fgcmLog.debug('FitCycle computing SuperStarFlats')
-        superStarFlat = FgcmSuperStarFlat(self.fgcmConfig,self.fgcmPars,self.fgcmStars)
-        superStarFlat.computeSuperStarFlats()
+        if not self.finalCycle:
+            self.fgcmLog.debug('FitCycle computing SuperStarFlats')
+            superStarFlat = FgcmSuperStarFlat(self.fgcmConfig,self.fgcmPars,self.fgcmStars)
+            superStarFlat.computeSuperStarFlats()
 
         if not self.quietMode:
             self.fgcmLog.info(getMemoryString('After computing superstar flats'))
 
         # Compute Aperture Corrections
-        self.fgcmLog.debug('FitCycle computing ApertureCorrections')
-        aperCorr = FgcmApertureCorrection(self.fgcmConfig,self.fgcmPars,self.fgcmGray)
-        aperCorr.computeApertureCorrections()
+        if not self.finalCycle:
+            self.fgcmLog.debug('FitCycle computing ApertureCorrections')
+            aperCorr = FgcmApertureCorrection(self.fgcmConfig,self.fgcmPars,self.fgcmGray)
+            aperCorr.computeApertureCorrections()
 
-        if not self.quietMode:
-            self.fgcmLog.info(getMemoryString('After computing aperture corrections'))
+            if not self.quietMode:
```

Comment:
You didn't indent `if not self.quietMode` above at ~L506, but you do here and below...deliberate?  Besides that, it seems these all could live within a single `if not self.finalCycle:` clause